src/elements/content-sidebar/__tests__/MetadataSidebarRedesigned.test.tsx

```diff
@@ -1,23 +1,75 @@
 import React from 'react';
-import { IntlProvider } from 'react-intl';
+import { screen } from '@testing-library/react';
+import MetadataSidebarRedesign from '../MetadataSidebarRedesign';
+import { FIELD_PERMISSIONS_CAN_UPLOAD } from '../../../constants';
+import { render } from '../../../test-utils/testing-library';
 
-import { screen, render } from '@testing-library/react';
+jest.unmock('react-intl');
+jest.unmock('lodash');
```

Comment:
is this being mocked somewhere that you have to unmock it?

Reply:
removed that